indico/MaKaC/webinterface/pages/contributionReviewing.py

```diff
@@ -123,12 +123,6 @@ def _getHeadContent(self):
                '\n'.join(['<script src="{0}" type="text/javascript"></script>'.format(url)
                           for url in self._asset_env['mathjax_js'].urls()])
 
-    def getCSSFiles(self):
-        return WPConferenceModifBase.getCSSFiles(self) + self._asset_env['contributions_sass'].urls()
-
-    def getJSFiles(self):
```

Comment:
Why would it be? The point is for `contributions_sass` to be included in `screen.scss`, so we don't need to serve extra bundles through `getCSSFiles`. Am I missing anything?